cocos/particle/burst.ts

```diff
@@ -111,7 +109,7 @@ export default class Burst {
             preFrameTime = (preFrameTime > 0.0) ? preFrameTime : 0.0;
             const curFrameTime = repeat(psys.time - psys.startDelay.evaluate(0, 1), psys.duration);
             if (this._curTime >= preFrameTime && this._curTime < curFrameTime) {
-                (psys as any).emit(this.count.evaluate(this._curTime / psys.duration, 1), dt - (curFrameTime - this._curTime));
+                psys.emit(this.count.evaluate(this._curTime / psys.duration, 1), dt - (curFrameTime - this._curTime));
```

Comment:
Don't convert to any since we're using Typescript, mark `emit` as public and engineInternal.